src/ServicePulse.Host/vue/tsconfig.app.json

```diff
@@ -3,6 +3,7 @@
   "include": ["env.d.ts", "src/**/*.js", "src/**/*.ts", "src/**/*.vue", "./test/**/*.ts"],
   "exclude": ["node_modules"],
   "compilerOptions": {
+    "moduleResolution": "Node",
```

Comment:
@cquirosj can you explain why we need to change this at all?

Reply:
https://github.com/Particular/ServicePulse/pull/1769/files#r1515342922